bazel/repository_locations.bzl

```diff
@@ -14,11 +14,11 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "thrift-0.11.0",
     ),
     com_github_bombela_backward = dict(
-        commit = "44ae9609e860e3428cd057f7052e505b4819eb84",  # 2018-02-06
+        commit = "84ae4f5e80381aca765a0810d4c811acae3cd7c7",  # v1.4
```

Comment:
Do you know if it's possible to switch to using the tags directly here? That would be easier to version bump and understand correspondence with.

Reply:
Yes. This library has releases and I'll update to
```
        sha256 = "ad73be31c5cfcbffbde7d34dba18158a42043a109e7f41946f0b0abd589ed55e",
        strip_prefix = "backward-cpp-1.4",        
        urls = ["https://github.com/bombela/backward-cpp/archive/v1.4.tar.gz"],
```